src/tank.ts

```diff
@@ -4,8 +4,8 @@ import { DirectionsType, Projectile } from './projectile';
 export class Tank {
   private _x: number;
   private _y: number;
-  private _rightBorder: number;
-  private _lowerBorder: number;
+  private readonly _rightBorder: number;
```

Comment:
Is this a kind of security measure?

Reply:
right, readonly prevents changing this field